com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfInertial.cs

```diff
@@ -96,24 +84,16 @@ public void UpdateLinkData()
 
         private void OnDrawGizmosSelected()
         {
+            // Do we need this? - ask Vidur; inertia tensor become a property later, and in 22.1 we have a nice viz too
```

Comment:
@vidurvij-Unity ^^

Reply:
I think we added this a long time ago in one of our debugging sessions. This is no longer needed.

Reply:
Ok got you, I'll cut this code then.